<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
    <!--<node name="lane_detection_sim" pkg="autovalet_lane_detection" type="erfnet_lane_detector.py" output="screen"/>-->

    <!-- Register depth image to rgb image -->
    <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

    <node pkg="nodelet" type="nodelet" name="nodelet1" args="load depth_image_proc/register nodelet_manager">
    <remap from="depth/camera_info" to="/frontCamera/depth/camera_info"/>
    <remap from="rgb/camera_info" to="/frontCamera/color/camera_info"/>
    <remap from="depth/image_rect" to="/frontCamera/depth/image_raw"/>
    </node>

  	<node name="lane_detection_sim" pkg="autovalet_lane_detection" type="autovalet_lane_detector_sim.py" output="screen"/>

</launch>
